AV-45749: Update sample queries (#35)
couchbaselabs/docs-devex: modules/guides/pages/transactions.adoc

include::partial$before-you-begin.adoc[tag=body]
include::partial$query-tools.adoc[tags=body;!thumbs]

> include::partial$data-warning.adoc[tag=page]

Reviewer: Not sure why you've made this change, since all the pages which include this partial only include the page warning? Forces the elixir branch to diverge from the other branches

Query Workbench::
+
--

> To specify parameters for a Couchbase transaction, use the Query Options window.

Reviewer 2: ```suggestion
To specify parameters for a Couchbase transaction, use the Query Options dialog.
```

> . To display the Query Options window, click btn:[options].

Reviewer 2: ```suggestion
. To display the Query Options dialog, click btn:[options].
```

> . Click the btn:[options] button to display the Query Options window.

Reviewer 2: ```suggestion
. To display the Query Options window, click btn:[options].
```

. To specify the transaction scan consistency, open the *Scan Consistency* drop-down list and select an option.

. To specify the transaction timeout, enter a value in seconds in the *Transaction Timeout* box.

> . To specify any other parameters, click the btn:[+] button in the corresponding *Named Parameters* or *Positional Parameters* section.

Reviewer 2: ```suggestion
. To specify any other parameters, click btn:[+] in the corresponding *Named Parameters* or *Positional Parameters* section.
```

** For named parameters, enter the name in the *name* box and a value in the *value* box.
** For positional parameters, enter the value in the *value* box.

> . Choose btn:[Save Settings] to save the preferences and return to the Query Workbench.

Reviewer 2: ```suggestion
. To save your preferences and return to the Query Workbench, click btn:[Save Settings].
```

'''

{lead}

> image::elixir-transaction-settings.png['The Query Options window, with Scan Consistency set to "not_bounded", Transaction Timeout set to "120", and named parameter "durability_level" set to "none"']

Reviewer 2: ```suggestion
image::elixir-transaction-settings.png['The Query Options dialog, with Scan Consistency set to "not_bounded", Transaction Timeout set to "120", and named parameter "durability_level" set to "none"']
```

Query Workbench::
+
--

> To execute a statement as a transaction, switch the *transaction* toggle on, enter the statement in the Query Editor and click btn:[Run Query].

Reviewer 2: ```suggestion
To execute a statement as a transaction, turn on the *transaction* toggle, enter the statement in the Query Editor and click btn:[Run Query].
```

include::n1ql:example$transactions/single.n1ql[tags=**]
----
--
endif::flag-devex-command-line[]
====

> For further information on executing transactions, see xref:n1ql:n1ql-language-reference/transactions.adoc#query-tools[Query Tools].

Reviewer 2: ```suggestion
For more information about executing transactions, see xref:n1ql:n1ql-language-reference/transactions.adoc#query-tools[Query Tools].
```

[[multiple-statement]]
== Multiple Statement Transactions

> // Multistatement transactions do not work in Elixir as of 2023/04/13 (SURF-4665)

Reviewer 3: now that's attention to detail :)

. Compose the sequence of statements in the Query Editor.
Each statement must be terminated with a semicolon.
// NOTE: SHift+Enter to start a newline does not work in Elixir as of 2023/04/13 (SURF-5251)
//. After each statement, press kbd:[Shift+Enter] to start a new line _without_ executing the query.

. When you have entered the entire transaction, click btn:[Run Query] to execute the transaction.

'''

{lead}

.Preparation
First specify the transaction settings, as shown in the section <<settings>> above.

Second, create a new keyspace for the transactions to run in:

> . Next to your bucket in the Data Insights sidebar, click the *More Options* (⋮) button, then btn:[Add Scope]. The Create Scope window now appears.

Reviewer 2: ```suggestion
. Next to your bucket in the Data Insights sidebar, click the *More Options* (⋮) button, then btn:[Add Scope].
```

. Enter the name "tenants" in the *Scope Name* field, then click the btn:[Create Scope] button.

> . Next to the new tenants scope, click the *More Options* (⋮) button, then btn:[Add Collection]. The Create Collection now appears.

Reviewer 2: ```suggestion
. Next to the new tenants scope, click the *More Options* (⋮) button, then btn:[Add Collection]. 
```
